.golangci.yml

```diff
@@ -132,6 +132,7 @@ linters:
     - godox
     - errcheck
     - testpackage
+    - godot
```

Comment:
why did we remove this linter?

Reply:
gen dir is auto generated and it cannot be changed it has some functions whose comments are not ended with dot (.) thats why i removed it

Reply:
Are we not ignoring generated code by excluding `gen` directory?